qa/smoke-test-ingest-with-all-dependencies/src/test/java/org/elasticsearch/ingest/IngestDocumentMustacheIT.java

```diff
@@ -31,7 +31,7 @@
 
 import static org.hamcrest.Matchers.equalTo;
 
-public class IngestDocumentMustacheIT extends AbstractMustacheTests {
+public class IngestDocumentMustacheIT extends AbstractMustacheTestCase {
```

Comment:
I think I can tighten up the checks a bit and it'd catch some more weirdness. I'll push something for that in a bit.